finance-tracker/app/views/portfolio/_result.html.erb

```diff
@@ -7,5 +7,17 @@
     <strong>Symbol: </strong> <%= @stock.ticker %>
     <strong>Name: </strong> <%= @stock.name %>
     <strong>Price: </strong> <%= @stock.last_price %>
+    <% if current_user.can_track_stock?(@stock.ticker) %>
+      <%= link_to 'Add to portfolio', user_stocks_path(user: current_user, ticker: @stock.ticker), class: "btn btn-xs btn-success", method: :post %>
+    <% else %>
+      <span class="badge badge-secondary">
+        You are already tracking
+        <% if current_user.stock_already_tracked?(@stock.ticker) %>
+          this stock
+        <% else !current_user.under_stock_limit? %>
```

Comment:
is this correct?
`else` or `elsif` ?

Reply:
shouldn't it be `else` since there are no other conditions?